Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Iterative_authalic_parameterizer_3.h

```diff
@@ -632,41 +631,33 @@ class Iterative_authalic_parameterizer_3
   /// \param mesh a triangulated surface.
   /// \param main_vertex_v_i the vertex of `mesh` with index `i`
   /// \param neighbor_vertex_v_j the vertex of `mesh` with index `j`
-  NT compute_w_ij(const Triangle_mesh& tmesh,
+  NT compute_w_ij(const Triangle_mesh& mesh,
                   vertex_descriptor main_vertex_v_i,
                   Vertex_around_target_circulator<Triangle_mesh> neighbor_vertex_v_j) const
   {
-    const PPM ppmap = get(vertex_point, tmesh);
-
-    const PPM_ref position_v_i = get(ppmap, main_vertex_v_i);
-    const PPM_ref position_v_j = get(ppmap, *neighbor_vertex_v_j);
+    const PPM ppmap = get(vertex_point, mesh);
+    const Point_3& position_v_i = get(ppmap, main_vertex_v_i);
+    const Point_3& position_v_j = get(ppmap, *neighbor_vertex_v_j);
 
-    // Compute the square norm of v_j -> v_i vector
-    Vector_3 edge = position_v_i - position_v_j;
-    NT square_len = edge*edge;
+    const Vector_3 edge = position_v_i - position_v_j;
+    const NT squared_length = edge * edge;
 
-    // Compute cotangent of (v_k,v_j,v_i) corner (i.e. cotan of v_j corner)
-    // if v_k is the vertex before v_j when circulating around v_i
     vertex_around_target_circulator previous_vertex_v_k = neighbor_vertex_v_j;
-    --previous_vertex_v_k;
-    const PPM_ref position_v_k = get(ppmap, *previous_vertex_v_k);
-//    NT cotg_psi_ij = internal::cotangent<Kernel>(position_v_k, position_v_j, position_v_i);
-    NT cotg_beta_ij = internal::cotangent<Kernel>(position_v_i, position_v_k, position_v_j);
+    previous_vertex_v_k--;
+    const Point_3& position_v_k = get(ppmap, *previous_vertex_v_k);
 
-    // Compute cotangent of (v_i,v_j,v_l) corner (i.e. cotan of v_j corner)
-    // if v_l is the vertex after v_j when circulating around v_i
     vertex_around_target_circulator next_vertex_v_l = neighbor_vertex_v_j;
-    ++next_vertex_v_l;
-
-    const Point_3 position_v_l = get(ppmap, *next_vertex_v_l);
-//    NT cotg_theta_ij = internal::cotangent<Kernel>(position_v_i, position_v_j, position_v_l);
-    NT cotg_alpha_ij = internal::cotangent<Kernel>(position_v_j, position_v_l, position_v_i);
-
-    NT weight = 0;
-    CGAL_assertion(square_len > NT(0)); // two points are identical!
-    if(square_len != NT(0))
-      weight = cotg_beta_ij + cotg_alpha_ij;
-
+    next_vertex_v_l++;
+    const Point_3& position_v_l = get(ppmap, *next_vertex_v_l);
+
+    NT weight = NT(0);
+    CGAL_assertion(squared_length > NT(0)); // two points are identical!
+    if (squared_length != NT(0)) {
```

Comment:
Please respect the indentation of the file in which you are modifying (when it's not broken).

Reply:
Do you mean the space after `if`? I did not think it is that important especially since I change the whole function. But I put back the old version. No pb.